coalib/output/ConsoleInteraction.py

```diff
@@ -684,7 +684,7 @@ def print_actions(console_printer, section, actions, failed_actions,
     if choice == 0:
         return None, None
 
-    return get_action_info(section, actions[choice - 1], failed_actions)
+    return get_action_info(section, actions[choice], failed_actions)
```

Comment:
because, `actions` list didn't had an action for `Do (N)othing`. When the user choose an action, Do (N)othing will be always on index 0.